app/components/press-release/form.js

```diff
@@ -53,6 +59,19 @@ export default class PressReleaseFormComponent extends Component {
     sources.removeObject(source);
   }
 
+  @action
+  async uploadFile(attachment) {
+    const attachments = await this.args.pressRelease.attachments;
+    attachments.pushObject(attachment);
+    this.showUploadModal = false;
+  }
+
+  @action
+  async removeAttachments(attachment) {
+    const attachments = await this.args.pressRelease.attachments;
+    attachments.removeObject(attachment);
```

Comment:
Na het verwijderen van de attachment uit de array ook de attachment zelf nog verwijderen (dmv `await attachment.destroyRecord()`) zodanig dat de opgeladen file in de backend ook verwijderd wordt.